### PR TITLE
vc: qemu: fix error message on hotplug.

### PR DESCRIPTION
**Pull request by jcvenegas**
The error message does not provide the max memory that is exceeded.

Fix it for better error information.

Fixes: #1120

Signed-off-by: Jose Carlos Venegas Munoz <jose.carlos.venegas.munoz@intel.com>

### CODE REVIEW

File: virtcontainers/qemu.go [hunk 1]
```diff
@@ -21,6 +17,11 @@ import (
 	"time"
 	"unsafe"
 
+	govmmQemu "github.com/intel/govmm/qemu"
```
Comment (grahamwhaley) [inline virtcontainers/qemu.go]: You probably should have put the import re-order in a separate commit ;-), then I'd not be typing in this little box :-)

**Comment by grahamwhaley** [comment]
Question - do we allow qemu to over-commit?

/test

**Comment by jcvenegas** [comment]
@grahamwhaley I have not tested how well it does with hotplug. Lets add a test for it.

File: virtcontainers/qemu.go [hunk 2]
```diff
@@ -1159,7 +1160,7 @@ func (q *qemu) hotplugMemory(memDev *memoryDevice, op operation) (int, error) {
 		if currentMemory+memDev.sizeMB > int(maxMem) {
 			// Fixme: return a typed error
```
Comment (jodh-intel) [inline virtcontainers/qemu.go]: Ugh - how did we miss this fixme previously? :(

**Comment by jodh-intel** [comment]
lgtm

**Comment by bergwolf** [comment]
LGTM

**Pull request merged**